deployment/test/test-transfer.ts

```diff
@@ -0,0 +1,593 @@
+import {
+  dependencies,
+  evm,
+  EvmChainInfo,
+  EvmScriptCb,
+  getContractAddress,
+  getEnv,
+  getEnvOrDefault,
+  LoggerFn,
+  SolanaChainInfo,
+  SolanaScriptCb,
+  TestTransfer,
+} from '../helpers';
+import { ethers } from 'ethers';
+import { getProvider, runOnEvms, sendTxCatch, wrapEthersProvider } from '../helpers/evm';
+import { SupportedChain, Tbrv3, Transfer } from '@xlabs-xyz/evm-arbitrary-token-transfers';
+import { resolveWrappedToken, toUniversal, UniversalAddress } from '@wormhole-foundation/sdk-definitions';
+import { Chain, chainIdToChain, chainToChainId, chainToPlatform, contracts } from '@wormhole-foundation/sdk-base';
+import { inspect } from 'util';
+import { solanaOperatingChains, getConnection, ledgerSignAndSend, runOnSolana, SolanaSigner } from '../helpers/solana';
+import { EvmAddress } from '@wormhole-foundation/sdk-evm';
+import { PublicKey, SystemProgram, TransactionInstruction } from '@solana/web3.js';
+import {
+  SolanaTokenBridgeRelayer,
+  TransferNativeParameters,
+  TransferWrappedParameters,
+} from '@xlabs-xyz/solana-arbitrary-token-transfers';
+import {
+  createAssociatedTokenAccountIdempotentInstruction,
+  createSyncNativeInstruction,
+  getAssociatedTokenAddress,
+  NATIVE_MINT
+} from '@solana/spl-token';
+
+import { ethers_contracts } from "@wormhole-foundation/sdk-evm-tokenbridge";
+import { deriveWrappedMintKey } from '@wormhole-foundation/sdk-solana-tokenbridge';
+
+
+const processName = 'att-test-transfer';
+const chains = evm.evmOperatingChains();
+const availableChains = chains.concat(solanaOperatingChains());
+const usdcAddresses = {
+  OptimismSepolia: '0x5fd84259d66Cd46123540766Be93DFE6D43130D7',
+  Sepolia: '0x1c7D4B196Cb0C7B01d743Fbc6116a902379C7238',
+  Celo: ' 0x2F25deB3848C207fc8E0c34035B3Ba7fC157602B',
+  Solana: '4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU'
+};
+
+/**
+ * This script expects the following environment variables:
+ * - WALLET_KEY: private key of the wallet to use for sending transactions
```

Comment:
Do we need a different env var to be able to send from evm and from solana in a single run?

Reply:
nvm, we can play with the skip flag to as well

Reply:
Hm, yeah, we probably do. I forgot to check that.

Reply:
I mean, it is possible to use the same private key but it's a bit inflexible.